code/script.js

```diff
@@ -1,19 +1,121 @@
 // Start here
 
 // Step 1 - Welcome and introduction
-// Your code goes here
-alert(
-  `Welcome to our Javascript Pizzeria. Ready to Start? - Click 'OK' to begin.`
-)
-
+alert(`Welcome to our Yia's Pizzeria. Ready to Start? - Click 'OK' to begin.`);
+let firstName = prompt("What is your name?");
+// alert("Hi " + firstName + "!");
+alert(`Hello ${firstName}!`);
+console.log(firstName);
 // Step 2 - Food choice
-// Your code goes here
+let orderFood = prompt(
+  "What type of food would you like to order? \nEnter your choice:\n1 - Pizza\n2 - Pasta\n3 - Salad"
```

Comment:
this is correct 👍 but have you tried though to use template literals for multiline text ``? Like so:
```
`What type of food would you like to order? 
Enter your choice:
1 - Pizza
2 - Pasta
3 - Salad`
```